packages/openactive-integration-tests/test/helpers/logger.js

```diff
@@ -223,24 +232,6 @@ class BaseLogger {
     else return "passed";
   }
 
-  get numValidationStatuses () {
```

Comment:
Is this being removed as it's unused?

Reply:
It is unused and it is a get accessor that returns no value, which TypeScript, by default, does not allow

Reply:
Could we just add `return result;` for now until we get more into this? I'm suspecting a bug here where it should be used but isn't...

Reply:
Sure. This is done